pythontest.py

```diff
@@ -42,71 +42,88 @@ def reading_file(file):
         # Reading csv and storing it into dataframe
         return pd.read_csv(file)
 
-
-origin = reading_file(file1)
-destination = reading_file(file2)
-
 def headerCheck(data):
     # Get all headers
     headers = data.columns
     required_columns = ['id', 'name', 'price']
     #checking headers on correct format
     for column in required_columns:
         if column not in headers:
-            sys.exit('The file does not have the right header columns. Please provide those columns in each file: id, name, price')
-
-headerCheck(origin)
-headerCheck(destination)
-
-#def file_copy(origin,destination):
-    # Creating first match criteria via ID
-    # Read all IDs of origin file and converting into regular array
-    #origin_ids = origin['id'].to_numpy()
-
-    # Read all IDs of destination file and converting into regular array
-    #destination_ids = destination['id']
-    # Looping through all origin ids and check if it exists in destination
-    # for id_value in destination_ids:
-    #     # If we find the id in the destination file we copy over the value
-    #     if id_value in origin_ids:
-    #         # Get the value of the origin id for the price
-    #         price = origin['price'][origin.id == id_value]
-    #         # Set value of destination file price to this price
-    #         destination['price'][destination.id == id_value] = price
-
+            sys.exit('The file does not have the right header columns. Please provide this column: ' + column)
 
 
 def file_copy(origin, destination):
     # iterate over ids in the destination file
+    # Set-up counter for matches
+    i = 0
```

Comment:
let's give these counters meaningful names, otherwise you loose track of what they contain